ci/compiler/source_manager.py

```
@@ -279,6 +279,24 @@ def copy_example_source(project_root: Path, build_dir: Path, example: str) -> bo
+    # is the path the user recognises if something is misdeclared.
+    try:
+        from ci.compiler.asset_scanner import (
+            announce_storage_requirements,
+            scan_sketch_assets,
+        )
+
+        announce_storage_requirements(scan_sketch_assets(example_path))
+    except KeyboardInterrupt as ki:
+        handle_keyboard_interrupt(ki)
+        raise
+    except Exception as exc:  # pragma: no cover - reporting must never break a build
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)